platform/windows/display_server_windows.cpp

```diff
@@ -1832,24 +1832,6 @@ void DisplayServerWindows::window_set_size(const Size2i p_size, WindowID p_windo
 
 	int w = p_size.width;
 	int h = p_size.height;
-
-	wd.width = w;
-	wd.height = h;
-
-#if defined(RD_ENABLED)
-	if (rendering_context) {
-		rendering_context->window_set_size(p_window, w, h);
-	}
-#endif
-#if defined(GLES3_ENABLED)
-	if (gl_manager_native) {
-		gl_manager_native->window_resize(p_window, w, h);
-	}
-	if (gl_manager_angle) {
-		gl_manager_angle->window_resize(p_window, w, h);
-	}
-#endif
```

Comment:
In the Windows GL manager implementation these functions do nothing, so it should work fine without it. But I think it should call it from `WM_WINDOWPOSCHANGED` anyway, just in case GL manager ever change, and for consistency with other platforms.

Reply:
Ah I see what you mean. Will add it